pkg/secrets_reflector/secrets_reflector.go

```diff
@@ -0,0 +1,153 @@
+package secretsreflector
+
+import (
+	"context"
+	"crypto/md5"
+	"encoding/base64"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/dcos/client-go/dcos"
+	log "github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+
+	dklbcache "github.com/mesosphere/dklb/pkg/cache"
+	"github.com/mesosphere/dklb/pkg/constants"
+)
+
+const (
+	defaultSecretStore = "default"
+	defaultTimeout     = 5 * time.Second
+)
+
+// SecretsReflector defines the interface exposed by this package.
+type SecretsReflector interface {
+	Reflect(namespace, name string) error
+}
+
+// DCOSSecretsAPI defines the interface required by the
```

Comment:
```suggestion
// DCOSSecretsAPI defines the interface required by the SecretsReflector to manage DC/OS secrets.
```

Reply:
I removed the hard wrap. It was done automatically by my code editor.